spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/Binding.java

```diff
@@ -37,4 +39,17 @@
 	 * and a new Binding should be created instead.
 	 */
 	void unbind();
+
+	/**
+	 * Binds the target component represented by this instance. Should be used to call any
+	 * lifecycle method on the represented target. Implementations must be idempotent.
+	 * This method is called after the concrete {@link Binder} implementation returns from
+	 * its create binding methods
+	 */
+	void bind();
+
+	/**
+	 * @return the binding target instance
+	 */
+	T getTarget();
```

Comment:
Could you elaborate the reasoning behind having this as part of the `Binding` contract itself? With this PR, I don't see `getTarget` used in a way (unless I miss something). Perhaps we need it in some other cases.

I am wondering if we can move this into `DefaultBinding` and not having this as part of the `Binding` interface?